.all-contributorsrc

```diff
@@ -172,15 +172,6 @@
       "contributions": [
         "code"
       ]
-    },
```

Comment:
Please don't remove contributors as the idea is to follow the all contributors standard.

Reply:
I'm sorry, I didn't mean to remove any contributors.